src/en/news/blog/2022/mclock-vs-wpq-testing-with-background-ops-part2/index.md

```diff
@@ -0,0 +1,271 @@
+As the balanced profile provides a reservation of 20% for background operations, it takes less time to complete scrub in comparison to the high recovery profile that makes minimum reservation (1) for background best effort.
+These results also coincide with what we saw in the previous sections where the high_client_ops profile has a lower average client latency and higher throughput than WPQ.
+To get a better understanding of how scrub looks like over the duration of the test, we can refer to the graph below.
+
+![](images/image6_scrub_rate_over_time.png)
+
+#### Experimenting with the mClock Custom Profile
+
+Our next step was to tune mClock profiles to finish scrubs faster than WPQ. The idea was to have an option that prioritizes scrubs over client operations if needed, similar to how the high_recovery_ops profile gives higher reservations for recovery operations.
+After several modifications to various mClock parameters, we found a profile with which scrub duration was lower than WPQ and the scrub rate was higher.
+The following are the modified parameters we used:
+
+|      Service Type      | Reservation | Weight | Limit |
+| :--------------------: | :---------: | :----: | :---: |
+|         client         |      1      |   1    |  80%  |
+|  background recovery   |     5%      |   1    | 100%  |
+| background best effort |     90%     |   2    |  MAX  |
+
+The graph below shows how the scrub looks over time in comparison to the default mClock profiles and WPQ.
+
+![](images/image7_scrub_rate_over_time_custom_profile_vs_WPQ.png)
+
+![](images/image8_scrub_time_vs_profile_with_custom_profile.png)
+
+From the graphs above we can clearly see that the custom profile is finishing scrub faster than WPQ and the high client profile.
+While this proved that mClock parameters can be tuned to give us the desired result, the custom profile is not a reasonable profile option for client and recovery options since they had to be strictly limited and have been given very low reservations.
+This led us to explore the existing scrub configuration parameters in Ceph. In the next section we take a look at how these parameters worked with WPQ and how we can tune them to work better with mClock and achieve lower scrub time without compromising client latency and throughput.
+
+#### Experimenting with Scrub Configuration Parameters
+
+The idea behind this experiment is to find out which existing scrub parameters can be tuned to finish scrub faster and how they affect client latency and throughput.
+In order to achieve lower scrub duration we looked into the following scrub configuration parameters:
+
+1. [osd_max_scrubs](https://docs.ceph.com/en/quincy/rados/configuration/osd-config-ref/#confval-osd_max_scrubs)
+2. osd_scrub_cost
```

Comment:
Maybe put a link for this config option here? https://github.com/ceph/ceph/blob/97d9de3340864c6ae55ba59cd7c8ea31f2a5cda0/src/common/options/global.yaml.in#L3607-L3614